user-service/src/main/java/org/java/mentorship/user/controller/SessionController.java

```diff
@@ -0,0 +1,41 @@
+package org.java.mentorship.user.controller;
+
+import lombok.AllArgsConstructor;
+import org.java.mentorship.contracts.user.dto.Session;
+import org.java.mentorship.contracts.user.dto.request.LoginRequest;
+import org.java.mentorship.user.service.SessionService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@AllArgsConstructor
+public class SessionController {
+    private final SessionService sessionService;
+
+    @PostMapping("/sessions")
+    public ResponseEntity<Session> createSession(@RequestBody LoginRequest loginRequest) {
```

Comment:
Shouldn't this param have @Valid as well?

Reply:
Validation should be done by the UserService which will throw an error and the gateway will catch it